x-pack/plugins/reporting/server/types.ts

```diff
@@ -8,15 +8,15 @@ import { KibanaRequest, RequestHandlerContext } from 'src/core/server';
 // eslint-disable-next-line @kbn/eslint/no-restricted-paths
 import { DataPluginStart } from 'src/plugins/data/server/plugin';
 import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
-import { SpacesPluginSetup } from '../../spaces/server';
-import { CancellationToken } from '../../../plugins/reporting/common';
 import { PluginSetupContract as FeaturesPluginSetup } from '../../features/server';
 import { LicensingPluginSetup } from '../../licensing/server';
 import { AuthenticatedUser, SecurityPluginSetup } from '../../security/server';
+import { SpacesPluginSetup } from '../../spaces/server';
```

Comment:
VS Code auto-import rules have definitely changed :crying_cat_face: 

Hopefully this type of noise can be disregarded.